upup/models/cloudup/resources/addons/networking.flannel/v0.7.0.yaml

```diff
@@ -0,0 +1,89 @@
+kind: ServiceAccount
+apiVersion: v1
+metadata:
+  name: flannel
+  namespace: kube-system
+---
+kind: ConfigMap
+apiVersion: v1
+metadata:
+  name: kube-flannel-cfg
+  namespace: kube-system
+  labels:
+    k8s-app: flannel
+data:
+  cni-conf.json: |
+    {
+      "name": "cbr0",
+      "type": "flannel",
+      "delegate": {
+        "forceAddress": true,
+        "isDefaultGateway": true
+      }
+    }
+  net-conf.json: |
+    {
+      "Network": "100.64.0.0/16",
+      "Backend": {
+        "Type": "udp"
+      }
+    }
+---
+kind: DaemonSet
+apiVersion: extensions/v1beta1
+metadata:
+  name: kube-flannel-ds
+  namespace: kube-system
+  labels:
+    k8s-app: flannel
+    role.kubernetes.io/networking: "1"
```

Comment:
All of the networking addons (weave, calico, kope.io) have this label in kops and so didnt see a reason not to include it. Not sure if there is a special kubernetes significance here. I can remove it if that makes sense?